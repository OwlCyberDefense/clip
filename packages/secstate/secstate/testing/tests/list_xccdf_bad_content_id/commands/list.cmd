/usr/sbin/secstate list foo
