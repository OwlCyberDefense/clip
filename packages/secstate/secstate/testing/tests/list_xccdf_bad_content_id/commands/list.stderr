No such file: 'foo'
Error loading benchmark: foo
